nautobot/dcim/elevations.py

```diff
@@ -107,8 +126,23 @@ def _draw_device_front(self, drawing, device, start, end, text):
     def _draw_device_rear(self, drawing, device, start, end, text):
         rect = drawing.rect(start, end, class_="slot blocked")
         rect.set_desc(self._get_device_description(device))
+
+        device_fullname = str(device)
+        device_shortname = settings.UI_RACK_VIEW_TRUNCATE_FUNCTION(str(device))
+
         drawing.add(rect)
-        drawing.add(drawing.text(str(device), insert=text))
+        drawing.add(
+            drawing.text(
+                device_fullname, insert=text, class_=f"rack-device-fullname{'' if self.display_fullname else ' hidden'}"
```

Comment:
`black` won't let me make both here the same presentation (`fullname` is shorter than `shortname`)